server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -40,8 +48,16 @@ export default class DashboardPresenter {
 
   readonly navItemsPresenter = new PrimaryNavBarPresenter('Referrals', this.loggedInUser)
 
-  readonly tableRows: SortableTableRow[] = this.referralsSummary.map(referralSummary => {
+  readonly tableRows: SortableTableRow[] = this.sentReferrals.content.map(referralSummary => {
     const sentAtDay = CalendarDay.britishDayForDate(new Date(referralSummary.sentAt))
+    const interventionForReferral = this.interventions.find(
```

Comment:
_not a blocker_; _performance_
💭 `find` here might do a few iterations until it finds the correct element; if we intend the `interventions` object to be a cache for this lookup, we could initialise it for this access, i.e. `{id1: Intervention1, id2: Intervention2}`

just flagging because UI is now leading in CPU consumption (blue is UI):
<img width="834" alt="image" src="https://user-images.githubusercontent.com/1526295/155129401-649e2631-615c-4b2d-b383-901a6b906aa5.png">